package.json

```diff
@@ -20,7 +20,7 @@
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
     "@bcrs-shared-components/corp-type-module": "1.0.9",
     "@bcrs-shared-components/court-order-poa": "2.1.4",
-    "@bcrs-shared-components/enums": "1.0.29",
+    "@bcrs-shared-components/enums": "1.0.33",
```

Comment:
I was hoping you'd update all the other dependencies here that increased due to shared enum changes.

Reply:
Sure, good idea, that's not a problem

Reply:
Thanks!